### DISCUSSION TITLE
Get Message history

### DISCUSSION THREAD

**Discussion by MoebusXIII** [Ludy87/xplora_watch#100] [General]
Hi,

I am using this integration and it is doing very a great job.
I was wondering if there is a way to get all push/get messages in Home Assitant?
If you could send it from HAS and get the answers as well that will be a massive improvement.

Let me know if that is already feasible and how to do it.
Thank you for your support
Moebus

**Comment by Ludy87** [comment]
Check: https://github.com/Ludy87/xplora_watch/issues/32

the API is a bit inflexible, it will take some time, but the watch can only write defined texts or emoji.

**Comment by Ludy87** [comment]
FYI: https://github.com/Ludy87/xplora_watch/releases/tag/v2.4.0

**Reply by MoebusXIII** [reply]
Thank you very much!!!
That works great
